app/assets/stylesheets/layouts.scss

```diff
@@ -2,14 +2,7 @@
 @import "fonts";
 
 
-.row {
-  margin: 10px 0px;
-  // clear: both;
-} /* .row */
-
-.shover {
```

Comment:
we used to have a "shover" div on the application layout because the navbar was fixed to the top. We don't need it anymore.